examples/cc3200/xively_demo/main.c

```diff
@@ -0,0 +1,1127 @@
+                            g_ucConnectionBSSID[3], g_ucConnectionBSSID[4],
+                            g_ucConnectionBSSID[5] );
+            }
+            memset( g_ucConnectionSSID, 0, sizeof( g_ucConnectionSSID ) );
+            memset( g_ucConnectionBSSID, 0, sizeof( g_ucConnectionBSSID ) );
+        }
+        break;
+
+        default:
+        {
+            UART_PRINT( "[WLAN EVENT] Unexpected event [0x%x]\n\r", pWlanEvent->Event );
+        }
+        break;
+    }
+}
+
+//*****************************************************************************
+//
+//! \brief This function handles network events such as IP acquisition, IP
+//!           leased, IP released etc.
+//!
+//! \param[in]  pNetAppEvent - Pointer to NetApp Event Info
+//!
+//! \return None
+//!
+//*****************************************************************************
+void SimpleLinkNetAppEventHandler( SlNetAppEvent_t* pNetAppEvent )
+{
+    if ( !pNetAppEvent )
+    {
+        return;
+    }
+
+    switch ( pNetAppEvent->Event )
+    {
+        case SL_NETAPP_IPV4_IPACQUIRED_EVENT:
+        {
+            SlIpV4AcquiredAsync_t* pEventData = NULL;
+
+            SET_STATUS_BIT( g_ulStatus, STATUS_BIT_IP_AQUIRED );
+
+            // Ip Acquired Event Data
+            pEventData = &pNetAppEvent->EventData.ipAcquiredV4;
+
+            // Gateway IP address
+            g_ulGatewayIP = pEventData->gateway;
+
+            UART_PRINT( "[NETAPP EVENT] IP Acquired: IP=%d.%d.%d.%d ,"
+                        "Gateway=%d.%d.%d.%d\n\r",
+                        SL_IPV4_BYTE( pNetAppEvent->EventData.ipAcquiredV4.ip, 3 ),
+                        SL_IPV4_BYTE( pNetAppEvent->EventData.ipAcquiredV4.ip, 2 ),
+                        SL_IPV4_BYTE( pNetAppEvent->EventData.ipAcquiredV4.ip, 1 ),
+                        SL_IPV4_BYTE( pNetAppEvent->EventData.ipAcquiredV4.ip, 0 ),
+                        SL_IPV4_BYTE( pNetAppEvent->EventData.ipAcquiredV4.gateway, 3 ),
+                        SL_IPV4_BYTE( pNetAppEvent->EventData.ipAcquiredV4.gateway, 2 ),
+                        SL_IPV4_BYTE( pNetAppEvent->EventData.ipAcquiredV4.gateway, 1 ),
+                        SL_IPV4_BYTE( pNetAppEvent->EventData.ipAcquiredV4.gateway, 0 ) );
+        }
+        break;
+
+        default:
+        {
+            UART_PRINT( "[NETAPP EVENT] Unexpected event [0x%x] \n\r",
+                        pNetAppEvent->Event );
+        }
+        break;
+    }
+}
+
+//*****************************************************************************
+//
+//! \brief This function handles HTTP server events
+//!
+//! \param[in]  pServerEvent - Contains the relevant event information
+//! \param[in]    pServerResponse - Should be filled by the user with the
+//!                                      relevant response information
+//!
+//! \return None
+//!
+//****************************************************************************
+void SimpleLinkHttpServerCallback( SlHttpServerEvent_t* pHttpEvent,

```

Comment:
This is a linker obligation... sorry - this is required by simplelink from SDK 1.2.0